.github/workflows/codeblocks_check.yml

```diff
@@ -0,0 +1,33 @@
+name: Validate Code::Blocks Project
```

Comment:
Combine this and the xcode workflow into the same workflow as different jobs in each.

We should consider updating the path filters (& possibly also the compute changes workflow) to support distinguishing when each should run.

Reply:
Should I use the existing compute-changes workflow or create a new one?

Reply:
The workflow file that defines the jobs for CodeBlocks & XCode projects would probably want to trigger based on all pushes that involve the game or test source directories, plus the respective IDE files/folders. That workflow (e.g. "IDE Validation") would have a job that calls the compute-changes workflow and the IDE specific jobs would consume its outputs to know if they should run.